src/rest.rs

```diff
@@ -104,14 +104,62 @@ impl BlockValue {
             #[cfg(not(feature = "liquid"))]
             nonce: header.nonce,
             #[cfg(not(feature = "liquid"))]
-            difficulty: header.difficulty(bitcoin::Network::from(network)),
+            difficulty: difficulty_new(header),
 
             #[cfg(feature = "liquid")]
             ext: Some(json!(header.ext)),
         }
     }
 }
 
+// The maximum target of bitcoin PoW as an f64
+// Current version of rust-bitcoin ignores the Network and always returned
+// the max target of bitcoin mainnet. Therefore the Network is irrelevant.
+const TARGET_MAX_F64: f64 = 2.695953529101131e67;
+
+fn difficulty_new(bh: &bitcoin::BlockHeader) -> f64 {
+    TARGET_MAX_F64 / u256_to_f64(bh.target())
+}
+
+#[inline]
+fn u256_to_f64(input: bitcoin::util::uint::Uint256) -> f64 {
```

Comment:
FYI this type will be hidden in the next release (soon), so after bumping the version this whole code has to be deleted. I suggest you just wait for the release (unless this is urgent and you can't spend time resolving *other* breakages).